scrapy/http/request/form.py

```diff
@@ -46,9 +59,25 @@ def from_response(cls, response, formname=None, formid=None, formnumber=0, formd
         formdata = _get_inputs(form, formdata, dont_click, clickdata, response)
         url = _get_form_url(form, kwargs.pop('url', None))
         method = kwargs.pop('method', form.method)
+
+        form_enctype = form.xpath('@enctype')
+        # update the header enctype and request multipart parameter
+        if len(form_enctype) and form_enctype[0]:
+            headers = kwargs.pop('headers', {})
```

Comment:
I think there is no need on using `kwargs` for getting headers, just specifying it as a method argument would do, and to not use the previous `if`, just get it with `form.xpath('@enctype').extract_first()`